src/dotnet/commands/dotnet-new/FSharp_Web/$projectName$.fsproj

```diff
@@ -10,20 +10,10 @@
     <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wp8+wpa81;</PackageTargetFallback>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <GlobalExclude>$(GlobalExclude);bin\**;obj\**;node_modules\**;jspm_packages\**;bower_components\**;**\*.user;**\*.*proj</GlobalExclude>
-  </PropertyGroup>
-
   <ItemGroup>
-    <None Include="**\*" />
     <Compile Include="Controllers\*.fs" />
     <Compile Include="Startup.fs" />
```

Comment:
The best solution ihmo is how Microsoft.NET.Sdk want to do that:

- web sdk should always add behavior with a conditional, like add compile *.cs with a conditional override `WebOverrideCompileInclude`
- other Lang like F# can disable it adding setting that property in the sdk package .props file


Meanwhile I can remove *.cs inside `Fsharp.NET.Sdk` later. (I'll publish another package but no dotnet/cli changes required)